app/views/find_local_council/one_council.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render layout: 'base_page', locals: {
+  title: 'Find Your Local Council'
+} do %>
+  <h2>Your postcode is in:</h2>
+
+  <div class="<%= @authority['tier'] %>-result group">
+    <p><%= @authority["name"] %></p>
+
+    <p>
+      <% if @authority["homepage_url"].blank? %>
+        We don't have a link for their website.
+      <% else %>
+        Website: <%= link_to extract_host(@authority["homepage_url"]), @authority["homepage_url"] %>
+      <% end %>
+    </p>
+  </div>
+  <hr class="separator">
+
+  <div class="search-again">
+    <a title="Search again using a different postcode." href="/find-your-local-council">Back</a>
```

Comment:
I think this may pose problems with screenreaders potentially? 

https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong